### PR TITLE
Rename --no-topic option of mqtt sub command as --brief

### PR DESCRIPTION
**Pull request by albinsuresh**
## Proposed changes

Rename `--no-topic` option of mqtt sub command as `--brief`

## Types of changes

What types of changes does your code introduce to `thin-edge.io`?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ x] New feature (non-breaking change which adds functionality)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Paste Link to the issue
<br/>

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ x] I have read the [CONTRIBUTING](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTING.md) doc
- [ x] I have signed the [CLA](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTOR-LICENSE-AGREEMENT.md) (in all commits with git commit -s)
- [x ] I ran `cargo fmt` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [ x] I used `cargo clippy` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [x ] I have added tests that prove my fix is effective or that my feature works
- [ x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: tedge/src/mqtt.rs [hunk 1]
```diff
@@ -34,9 +34,9 @@ pub enum MqttCmd {
         /// QoS level (0, 1, 2)
         #[structopt(short, long, parse(try_from_str = parse_qos), default_value = "0")]
         qos: QoS,
-        /// Avoid printing the message topics on the console
-        #[structopt(long = "no-topic")]
-        no_topic: bool,
+        /// Print only the MQTT message to the console and not the topic
+        #[structopt(short, long)]
+        brief: bool,
```
Comment (ChrisGreenaway) [inline tedge/src/mqtt.rs:39]: Personally I prefer "no-topic" rather than "brief".

Comment (didier-wenzek) [reply tedge/src/mqtt.rs:39]: Agree with Chris.

For instance for grep.
```
       -H, --with-filename
              Print the file name for each match.  This is the default when there is more than one file to search.

       -h, --no-filename
              Suppress the prefixing of file names on output.  This is the default when there is only one file (or only standard input) to search.

```

Comment (mneumann) [reply tedge/src/mqtt.rs:39]: I prefer `brief` or `verbose`. `no_topic` is confused when you have a `--topic abc` option. You end up with:

```sh
> tedge sub --topic abc --no-topic
```

Comment (mneumann) [reply tedge/src/mqtt.rs:39]: Or `--show-topic` or `--hide-topic`.

Comment (rina23q) [reply tedge/src/mqtt.rs:39]: `tedge sub` doesn't get a topic as an option. `tedge sub <topic>` is expected.
But I vote for `--hide-topic`. Or, `--no-print-topic`.

**Pull request closed without merging**